src/MarkerGeneHeatmap.js

```diff
@@ -242,7 +242,7 @@ const MarkerGeneHeatmap = (props) => {
     exporting: {
       buttons: {
         contextButton: {
-          text: '<i class="icon icon-functional" data-icon="="></i> Download &nbsp;',
+          text: '<i class="icon icon-functional" data-icon="="></i>Download&nbsp;',
```

Comment:
The space should be **between** the `<i>` element and the *Download* label: `<i ...></i>&nbsp;Download`